app/src/router.js

```diff
@@ -21,7 +21,7 @@ export const routeGuard = (store) => async (to, from, next) => {
     if (store.state.account.userSignedIn) {
       next()
     } else {
-      next(`/paywall`)
+      next(`/notification-wall`)
```

Comment:
?? :eyes:

Reply:
You would need to change the routes thing too no?

Reply:
ah it works :shrug: Magic

Reply:
i did in a previous PR - but forgot this one link that still referenced `/paywall`

Reply:
shouldn't it be `notifications-wall` though? In plural